dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/metric_config_panel_item.tsx

```diff
@@ -0,0 +1,238 @@
+                      />
+                    </EuiText>
+                  )
+                }
+              >
+                <EuiComboBox
+                  aria-label="Accessible screen reader label"
+                  placeholder="Select a aggregation"
+                  singleSelection={false}
+                  options={METRICS_AGGREGATION_OPTIONS}
+                  selectedOptions={
+                    singleField.aggregation.length > 0 ? singleField.aggregation : []
+                  }
+                  onChange={(e) =>
+                    updateListWithAgg(e.length > 0 ? e : '', index, sectionName, 'aggregation')
+                  }
+                />
+              </EuiFormRow>
+              <EuiFormRow label="Field">
+                <EuiComboBox
+                  aria-label="Accessible screen reader label"
+                  placeholder="Select a field"
+                  singleSelection={{ asPlainText: true }}
+                  options={getOptionsAvailable(sectionName)}
+                  selectedOptions={singleField.label ? [{ label: singleField.label }] : []}
+                  onChange={(e) =>
+                    updateList(e.length > 0 ? e[0].label : '', index, sectionName, 'label')
+                  }
+                />
+              </EuiFormRow>
+
+              <EuiFormRow label="Custom label">
+                <EuiFieldText
+                  placeholder="Custom label"
+                  value={singleField.custom_label}
+                  onChange={(e) => updateList(e.target.value, index, sectionName, 'custom_label')}
+                  aria-label="Use aria labels when no actual label is in use"
+                />
+              </EuiFormRow>
+
+              <EuiSpacer size="s" />
+              {visualizations.vis.name !== visChartTypes.HeatMap && lists.length - 1 === index && (
+                <EuiFlexItem grow>
+                  <EuiButton
+                    fullWidth
+                    iconType="plusInCircleFilled"
+                    color="primary"
+                    onClick={() => handleServiceAdd(sectionName)}
+                  >
+                    Add
+                  </EuiButton>
+                </EuiFlexItem>
+              )}
+            </EuiPanel>
+          </div>
+        </div>
+        <EuiSpacer size="s" />
+      </>
+    ));
+
+  return (
+    <>
+      <EuiTitle size="xxs">
+        <h3>Data Configurations</h3>
+      </EuiTitle>
+      <EuiSpacer size="s" />
+      <EuiSpacer size="s" />
+      <EuiTitle size="xxs">
+        <h3>Metrics</h3>
+      </EuiTitle>
+      {getCommonUI(configList.metrics, 'metrics')}
+      <EuiFlexItem grow={false}>
+        <EuiButton
+          data-test-subj="visualizeEditorRenderButton"
+          iconType="play"
+          onClick={updateChart}
+          size="s"
+        >
+          Update chart
+        </EuiButton>
+      </EuiFlexItem>
+    </>
+  );
+};
```

Comment:
We have changed this file. Please update the config list according to valueOptions